packages/ui/src/components/Accordion/Accordion.tsx

```diff
@@ -134,13 +139,22 @@ export const Accordion = (props: AccordionProps) => {
         {...summaryProps}
         data-qa-panel-summary={heading}
       >
-        <Typography {...headingProps} data-qa-panel-subheading variant="h3">
-          {heading}
-          {headingChip}
-        </Typography>
-        {headingNumberCount && headingNumberCount > 0 ? (
-          <span className={classes.itemCount}>{headingNumberCount}</span>
-        ) : null}
+        <div
+          style={{ display: 'flex', flexDirection: 'column', rowGap: '6px' }}
+        >
+          <Typography {...headingProps} data-qa-panel-subheading variant="h3">
+            {heading}
+            {headingChip}
+          </Typography>
+          {subHeading ? (
+            <Typography data-qa-panel-subheading variant="body1">
+              {subHeading}
+            </Typography>
+          ) : undefined}
```

Comment:
```suggestion
          {subHeading && (
            <Typography color="textSecondary" variant="body1">
              {subHeading}
            </Typography>
          )}
```

Also add `color="textSecondary"` here